tsl/test/sql/continuous_aggs_ddl.sql

```diff
@@ -734,5 +734,31 @@ ALTER MATERIALIZED VIEW owner_check OWNER TO :ROLE_1;
 SELECT * FROM cagg_info WHERE user_view::text = 'owner_check';
 \x off
 
+--
+-- Test DROP SCHEMA CASCADE with continuous aggregates
+--
+-- Issue: #2350
+
+CREATE SCHEMA test_schema;
+
+CREATE TABLE test_schema.telemetry_raw (
+  ts        TIMESTAMP WITH TIME ZONE NOT NULL,
+  value     DOUBLE PRECISION
+);
+
+SELECT create_hypertable('test_schema.telemetry_raw', 'ts');
+
+CREATE MATERIALIZED VIEW test_schema.telemetry_1s
```

Comment:
could you add 1 more mat view for the test case so that we have multiple caggs attached to the hypertable?